schedule/containers/engines_and_tools-s390x.yaml

```diff
@@ -0,0 +1,35 @@
+name:           engines_and_tools-s390x
+description:    >
+  Maintainer: qa-c@suse.de.
+  This schedule is focused on testing the related container packages and features,
+  but not the official container images from SUSE and openSUSE.
+conditional_schedule:
+  supported:
```

Comment:
```suggestion
  buildah_podman:
```

Reply:
I would keep it as `supported` as also `registry` should be there as that test module is also not executed on every `VERSION`.